packages/fxa-settings/src/components/Settings/AccountRecoveryWizard/index.tsx

```diff
@@ -0,0 +1,133 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { FlowContainer } from '../FlowContainer';
+import { ProgressBar } from '../ProgressBar';
+
+export const AccountRecoveryWizard = (props: RouteComponentProps) => {
+  const [currentStep, setCurrentStep] = useState(1);
+  /*
+    Making this a component (as opposed to writing it directly into the PageRecoveryAdd view) will hopefully allow us to separate the logic a little more from the UI.
+    In this wizard pattern, you'll want to have equality checks which control which step is being displayed (like,
+      { currentStep === 1 && (<YourViewHere/> )}
+    ).
+    Each step will be built with the <FlowContainer /> component as the outer wrapper.  Each step will at minimum expect the props `currentStep` and `setCurrentStep`, received from this component. The `currentStep` value and `setCurrentStep` method will be passed in to the `<FlowContainer/>` as props of the same name. Each step will want to have the <ProgressBar /> component as a child, and the title will be 'Account recovery key'.
+    We have filler components hardcoded in here (the Wizard pattern seems too simple to justify making it into a reusable component), but we can replace these as we create the real views.
+    Each step should be a separate component.
+  */
+  return (
+    <>
+      {
+        // Create an account recovery key in case you forget your password
+        currentStep === 1 && (
+          <FlowContainer
+            title="Account recovery key"
```

Comment:
Q: Does the `title` get localized?
(and same comment re: `<h2>` / `<p>` / `<button>` tags below.)

Reply:
Same as comments below